contivctl/commands.go

```diff
@@ -0,0 +1,191 @@
+package contivctl
+
+import (
+	"github.com/codegangsta/cli"
+)
+
+var tenantFlag = cli.StringFlag{
+	Name:  "tenant, t",
+	Value: "default",
+	Usage: "Name of the tenant to create the policy on.",
+}
+
+var Commands = []cli.Command{
+	{
+		Name:  "group",
```

Comment:
sorry for nitpicking: should we call 'group' or 'policy-domain' or 'endpoint group' because containers can belong to two different groups at a given time.